frontend/src/features/comparisonSeries/ComparisonSeries.tsx

```diff
@@ -87,7 +89,9 @@ const ComparisonSeries = ({
    * series.
    */
   useEffect(() => {
-    async function getAlternativesAsync(getAlts: () => Promise<Array<Entity>>) {
+    async function getAlternativesAsync(
+      getAlts: () => Promise<Array<Entity | Recommendation>>
```

Comment:
You wrote many times `Entity | Recommendation`, do you think it is worth it to define this once? It would get more complex if we get a third wrapper around entity objects.

Another option maybe to keep using the more restricted type `Entity` here and convert Recommendation to Entity ?

Reply:
Good question.

I didn't convert the `Recommendation` to `Entity` on purpose. First because the tutorial must work with both entities (it's the présidentielle use case), and recommendations (videos use case). It's what I expect from it, to be able to interact with both API. Keeping these two types make the behaviour of the code more explicit.

Second because I think we may be able to simplify the tutorial by working only with UID, and not directly with entities and recommendations. I think we can avoid more complexity by not using casting functions.

About the possible wrapper around those two types. Usually in the front end, dealing with a variable that can have many types introduce more complexity. We are not sure all desired attributes are available, we must test their existence, etc. etc. I think I'd like to simplify the tutorial by using only UIDs, before thinking about adding a wrapper.

What do you think?

Reply:
Note that `RelatedEntityObject` defined in "src/utils/types.ts" could probably be used here instead of `Entity | Recommendation`. 

(Arguably, we might want to rename `RelatedEntityObject` to a more expressive name, such as `GenericEntity` or `EntityObject`)